modules/ROOT/pages/aurads/connecting/neo4j-applications.adoc

```diff
@@ -94,7 +94,7 @@ To connect to an AuraDS instance using Neo4j Desktop:
 
 . Navigate to the https://console.neo4j.io/?product=aura-ds[Neo4j Aura Console^] in your browser.
 . Copy the *Connection URI* of the instance you want to connect to.
-The URI is in the page that opens when clicking on the instance.
```

Comment:
the line also appears in the Cypher Shell section, it should be fixed there too

Reply:
Yes sir! Will do!